### PR TITLE
Energy Efficiency Directive documentation (TEDEFO-3658)

### PR DESCRIPTION
**Pull request by YvesJo**

### CODE REVIEW

**Review by bertrand-lorentz — CHANGES_REQUESTED** [review]

File: modules/schema/pages/procedure-lot-part-information.adoc [hunk 1]
```diff
@@ -1021,6 +1036,49 @@ This information is also possible at Lot level for Result notices, especially
...
+
+The Energy Efficiency Directive (EED) Applicable Business Term is used to specify whether this 
+directive applies or not. When used, it should contain either of the two 
+values "true" or "false".
+When BT-810 exists and has its value set to "true", the EED List (item) (BT-811) 
+must also be specified, at least once, using one of the values of the https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/energy-efficiency-item[Energy Efficiency Item] 
+codelist. This BT is repeatable.
+
+A sample mark-up is provided below for a case where the directive applies 
+and procurement concerns "Power transformers" as well as "Water pumps" (the structure occurs in the extensions of the TenderingTerms of
```
[the remark this answers is not in the document]
Comment (pdonohoe) [reply modules/schema/pages/procedure-lot-part-information.adoc:1058]: I suggest:
procurement concerns "Dishwashers" as well as "Light sources and control gears"

**Comment by pdonohoe** [comment]
The A+++ etc do not display correctly for me. The line:
|*Energy Efficiency Label (BT-812)* |Energy efficiency labels of the provided products (A+++, A++, A+, A to G).  
Displays for me as: 
Energy efficiency labels of the provided products (A+, A, A+, A to G).

I don't know how to display a sequence of + symbols

**Comment by pdonohoe** [comment]
&#43; can be used to display a + symbol. This works for me:
|*Energy Efficiency Label (BT-812)* |Energy efficiency labels of the provided products (A&#43;&#43;&#43;, A&#43;&#43;, A&#43;, A to G).

**Comment by pdonohoe** [comment]
I added the change suggested by Bertrand, and my suggested changes. Also changed "Union GPP" to "European Union GPP" and changed the description for "Energy Efficiency Quantity (BT-815)" to: 
The quantity provided or estimated. For products, the number of products. For buildings, the useful floor area in m2. For works or services the value will usually be 1.
This is taken from the Guide document. It still sounds clumsy, but I'm not sure how to improve it.

**Review by pdonohoe — APPROVED** [review]

**Pull request merged**